addons/web/static/tests/views/form_tests.js

```diff
@@ -415,8 +415,6 @@ QUnit.module('Views', {
                 // field, not sure why.  Maybe this test should be modified.
                 assert.deepEqual(args.args[1], ["foo", "display_name"],
                     "should only fetch requested fields");
-                assert.deepEqual(args.kwargs.context, {bin_size: true},
```

Comment:
you could keep this I think... By default, bin_size is set to true in the context